lib/fhir_client.rb

```diff
@@ -1,26 +1,23 @@
-# Top level include file that brings in all the necessary code
-require 'bundler/setup'
-require 'yaml'
-require 'nokogiri'
+require 'fhir_client/version'
+
+# require 'yaml'  # not sure what (if anything) this is needed for
```

Comment:
ah ok.  Nokogiri is still there, it's just been moved.

My original comment was about the 'yaml' dependency.